degica/knifeswitch
# Rails 6 support

The library worked fine; just needed to change the testing infrastructure and gemspec a bit.

## How to test

`bundle install` - confirm that it installs rails 6.0

If you have docker (recommended): `bin/dockertest`

If you don't have docker: `cd test/dummy && rake knifeswitch:create_migrations db:create db:migrate; bin/test`

NOTE: Since this project is all about timing, the tests can be flaky when your computer slows down.
